contracts/RewardPool.sol

```diff
@@ -92,65 +93,112 @@ contract RewardPool is Initializable, Versionable, Ownable {
     return true;
   }
 
+  function valid(
+    bytes memory leaf,
+    bytes32[] memory proof
+  ) public view returns (bool) {
+        (
+      address rewardProgramID,           // solhint-disable-line no-unused-vars
```

Comment:
actually I think it might be better if only for slightly reduce gas consumption not assigning those vars, but maybe the compiler optimises it out anyway

Reply:
Yeah, had tried to mark them as unused in other ways but hadn't found one that worked, this does thanks! I initially tried adding in the commented variables to make it less likely something pulls the wrong part of the data, but the formatter pulls them out into weird places so have gone with the first format.